core/blockchain.go

```diff
@@ -2817,7 +2817,12 @@ func (bc *BlockChain) GetTransactionLookup(hash common.Hash) *rawdb.LegacyTxLook
 func (bc *BlockChain) Config() *params.ChainConfig { return bc.chainConfig }
 
 // QuorumConfig retrieves the Quorum chain's configuration
-func (bc *BlockChain) QuorumConfig() *QuorumChainConfig { return bc.quorumConfig }
+func (bc *BlockChain) QuorumConfig() *QuorumChainConfig {
+	if bc == nil || bc.quorumConfig == nil {
+		return &QuorumChainConfig{}
```

Comment:
Why not returning nil (as before)?

Reply:
if you prefer we can change the chain_makers.go to avoid this:
```
func (b *BlockGen) AddTx(tx *types.Transaction) {
	b.AddTxWithChain(&BlockChain{quorumConfig: &QuorumChainConfig{}}, tx)
}
```

Reply:
or it's was a flacky unit test when I try, a solution could be like this:
```
func (b *BlockGen) AddTx(tx *types.Transaction) {
	b.AddTxWithChain(&BlockChain{quorumConfig: &QuorumChainConfig{}}, tx)
}
```